### ISSUE TITLE
Allow using `$EDITOR` to set file status

### ISSUE THREAD

**Issue by lenormf** [rndusr/stig#143]
Hi,

It'd be nice to have a key binding that sends the current fileslist to `$EDITOR`, so that we can use an external program to set the priorities individually. Think `git rebase -i`.

Thanks!

**Comment by rndusr** [comment]
That's a good idea. Thanks.

Not sure if it's going to be in the next release. PRs are welcome.

**Comment by lenormf** [comment]
Quick question, do you care about compatibility with Windows?

**Comment by rndusr** [comment]
Not really. I try to keep things cross-platform where I can, but I've never
tested it.

I haven't heard from any Windows users, which probably means they don't exist.

But there are probably Mac users.

**Comment by Kabouik** [comment]
*Sees lenormf asking for `$EDITOR` support. Starts imagining using `kak` in `stig` too.* :eyes:
